ui/src/pages/ShoeMap.tsx

```diff
@@ -89,8 +89,34 @@ const StyledLogo = styled.div`
 export const ShoeMap: React.FC = () => {
   const zoom = 4;
   const minZoom = 4;
-  const maxZoom = 18;
+  const maxZoom = 12;
+  const timeoutSeconds = 1728000000;
   const currentLocation = { lat: 53.655697, lng: -100.13316 };
+  const key = "timestamp";
+
+  const setTimeStamp = () => {
+    const item = {
+      value: "value",
+      expiry: new Date().getTime(),
+    };
+    localStorage.setItem(key, JSON.stringify(item));
+  };
+
+  const compareTimestamp = () => {
```

Comment:
Let's move this func and `setTimeStamp` out of the component

Reply:
Moved them into a separate file to make it easier to test